shared/chat/conversation/list/index.native.js

```diff
@@ -76,7 +76,6 @@ class ConversationList extends Component<void, Props, void> {
         renderScrollComponent={this._renderScrollComponent}
         onEndReached={this._onEndReached}
         onEndReachedThreshold={0}
-        initialNumToRender={30}
```

Comment:
use the default of 10 instead. using 30 perma-mounts these items and its expensive.https://github.com/facebook/react-native/blob/c848c3820b21205f66f5756e29ba38b17ec994d0/Libraries/Lists/FlatList.js#L122